packages/react-api/package.json

```diff
@@ -10,5 +10,8 @@
     "@polkadot/extension-dapp": "^0.38.5",
     "@polkadot/x-rxjs": "^6.9.1",
     "fflate": "^0.7.0"
+  },
+  "devDependencies": {
```

Comment:
This should be a proper dependency not a dev dependency

Reply:
Done. Thank you